src/pages/Landing/components/EventsCalendarPopover.tsx

```diff
@@ -24,13 +24,17 @@ const EventsCalendarPopover = ({ id }: EventsCalendarPopoverProps) => {
     <Stack component={Paper} gap={1} sx={{ minWidth: 210, py: 1, px: 2 }}>
       {data && (
         <>
-          <DetailsHeader variant='h2'>Detaljer</DetailsHeader>
+          <DetailsHeader variant='h2'>{data.title}</DetailsHeader>
           <DetailContent info={formatDate(parseISO(data.start_date))} title='Fra:' />
           <DetailContent info={formatDate(parseISO(data.end_date))} title='Til:' />
           <DetailContent info={data.location} title='Sted:' />
-          <DetailsHeader variant='h2'>Påmelding</DetailsHeader>
-          <DetailContent info={`${data.list_count}/${data.limit}`} title='Påmeldte:' />
-          <DetailContent info={String(data.waiting_list_count)} title='Venteliste:' />
+          {data.sign_up && (
+            <>
+              <DetailContent info={`${data.list_count}${data.limit > 0 && '/' + data.limit}`} title='Påmeldte:' />
```

Comment:
Foreslår å unngå å bruke && for conditional rendering! 
https://dev.to/maafaishal/avoid-operator-for-conditional-rendering-in-react-2de

Reply:
Helt enig at det er bad når vi ikke vet hva `data.sign_up` er, men her vet vi at det er en av type boolean.

Reply:
Godt poeng. Ville unngått det uansett siden det er fort gjort å bruke det andre steder (også kan det hende at andre ser det og tenker at det kan brukes overalt). Men koden funker jo fint, så det er opp til deg 🫡